config/firstalert/zcombo-g.xml

```diff
@@ -0,0 +1,45 @@
+<Product Revision="1"
+  xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="Name">Battery Smoke/CO Combo Alarm</MetaDataItem>
+    <MetaDataItem name="Description">ZWave enabled, battery-powered combination smoke and carbon monoxide alarm</MetaDataItem>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/0138:0003:0001</MetaDataItem>
+    <MetaDataItem name="ProductPage">https://products.z-wavealliance.org/products/3886</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/firstalert/zcombo-g.png</MetaDataItem>
+    <MetaDataItem name="ProductManual">https://products.z-wavealliance.org/ProductManual/File?folder=&amp;filename=product_documents/3886/User_Manual_M08-0456-173833_D2.pdf</MetaDataItem>
+    <MetaDataItem name="WakeupDescription">To manual wake-up the device, slide the battery door open wait for ~5 seconds and then slide the battery drawer closed. Upon power-up, the device will send the wake-up notification.</MetaDataItem>
+    <MetaDataItem name="InclusionDescription">1. Slide battery door open.
+2. Insert batteries checking the correct orientation.
+3. Press and hold the test button. Keep it held down as you slide the battery drawer closed. You may then release the button.</MetaDataItem>
+    <MetaDataItem name="ExclusionDescription">1. Slide battery door open.
+2. Insert batteries checking the correct orientation.
+3. Press and hold the test button. Keep it held down as you slide the battery drawer closed. You may then release the button.</MetaDataItem>
+    <MetaDataItem name="ResetDescription">If the device is powered up with the test button held down for 10+ seconds, the device will reset all Z-Wave settings and leave the network.
+Upon completion of the Reset operation, the LED will glow and the horn will sound for ~1 second.
+Please use this procedure only when the network primary controller is missing or otherwise inoperable.
+    </MetaDataItem>
+    <MetaDataItem id="0003" name="ZWProductPage" type="0001">https://products.z-wavealliance.org/products/3886/</MetaDataItem>
+    <MetaDataItem id="0003" name="FrequencyName" type="0001">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0003" name="Identifier" type="0001">ZCOMBO-G</MetaDataItem>
```

Comment:
The manufacturer's data here https://products.z-wavealliance.org/products/3886/xml
for this device is showing the following values:
```
  <Identifier>ZCOMBO</Identifier>
  <Name>ZCOMBO-G</Name>
```

I suggest that in order to align with these values, this PR will have:
```
<MetaDataItem name="Name">ZCOMBO-G Battery Smoke/CO Alarm</MetaDataItem>
<MetaDataItem id="0003" name="Identifier" type="0001">ZCOMBO</MetaDataItem>
```